src/transport/websocket.ts

```diff
@@ -430,22 +429,22 @@ export default class WebSocketTransport implements SubscriptionTransport {
   private onEventMessage(
     eventMessage: Message,
     subscriptionListeners: SubscriptionListeners,
-  ): Error | void {
+  ) {
     if (eventMessage.length !== 3) {
-      return new Error(
+      new ProtocolError(
```

Comment:
How does this work without the `return`? The method body is not a single `if` expression...